examples/nft-nd-nns/nns.yml

```diff
@@ -14,4 +14,4 @@ events:
       - name: tokenId
         type: ByteArray
 permissions:
-  - methods: ["onNEP11Payment"]
+  - methods: ["onNEP11Payment", "update"]
```

Comment:
These better be different, it can call `onNEP11Payment` in any contract, but `update` in one specific.